docs/PERMISSIONS.md

```diff
@@ -57,9 +57,9 @@ Mizu needs following permissions on your Kubernetes cluster to run properly
   - get
 ```
 
-## Permissions required for service / pod name resolving (opt)
+## Permissions required for service / pod name resolving
```

Comment:
Permissions required running with --daemon flag or (optional) for service / pod name resolving in non daemon mod